.github/data/proxy_config.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
what's this change?  Did'ja just hate that LineFeed?

No trouble here just curiosity

Reply:
I guess. I had modified it locally and when I reverted I guess I removed the line.